libafl/src/stages/tmin.rs

```diff
@@ -93,20 +101,21 @@ where
             }
 
             let mut next_i = i + 1;
-            let mut input = base.clone();
+            let mut input_transformed = transformed.clone();
 
-            let before_len = input.len();
+            let before_len = base.len();
```

Comment:
shouldn't this be input_transformed.len()?

Reply:
This is one of the awkward places.

The `input_transformed` is now of the wrapped type `MutatedTransform<Input, State>`. I think here we want the length of the internal input (if I understand correctly). If we use `try_transform_into` to get the internal input (so that we can call `.len()`), `MutatedTransform<Input, State>` will be consumed. But we still need to keep `MutatedTransform<Input, State>` to pass into the mutator.